sonar-project.properties

```diff
@@ -9,5 +9,5 @@ sonar.sources.exclusions=**/*_test.go,**/*.spec.ts,**/*.spec.tsx
 sonar.test.inclusions=**/*_test.go,**/*.spec.ts,**/*.spec.tsx
 sonar.sourceEncoding=UTF-8
 sonar.go.coverage.reportPaths=coverage.out
-sonar.javascript.lcov.reportPaths=web/lcov.info
+sonar.javascript.lcov.reportPaths=web/coverage/lcov.info
```

Comment:
This was not a mistake. The github workflow extracts all artifacts to /web so the lcov.info file ends up in /web/lcov.info instead of the coverage folder.

Reply:
The upload artifacts action seems to preserve the folder hierarchy though, so `web/coverage/lcov.info` gets stored as `coverage/lcov.info`, which is then  downloaded to the `web` folder. You can see it if you download the `frontend-tests.zip` here: https://github.com/keydrive/keydrive/actions/runs/1082079441

Reply:
Huh, then I wonder why sonar was crashing when I set this up. You’re right that the upload artifact action preserves file paths. But the download artifact action did not so sonar kept complaining about the lcov file not existing. But it looks okay now I think? Hard to test on external servers…